update_diagnostics.py

```
@@ -0,0 +1,452 @@
+import re
+
+with open("crates/logfwd-io/src/diagnostics.rs", "r") as f:
```

Ruff (0.15.9)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)